### DISCUSSION TITLE
X / Twitter threads

### DISCUSSION THREAD

**Discussion by codyw912** [karakeep-app/karakeep#688] [Ideas]
I know it's non trivial, but being able to save and unfurl a Twitter / X thread would be very nice to have here. It's the one thing Readwise Reader does really well, but none of the open source apps I've tried have really nailed yet.

Do you have any thoughts on ways this could be done?

**Comment by MohamedBassem** [comment]
Twitter is pretty annoying in crawling. When you're logged out, it only shows you the tweet and nothing more. I think it's something that will get better once we have #172.

**Reply by codyw912** [reply]
Totally, they make it quite annoying. The solution proposed in #172 makes sense for web, but probably doesn't help on mobile. Better than nothing for sure.

I wonder if there's a potential solution on the server side.

**Comment by akshara-tg** [comment]
I have been using this site, it's a closed source, but it does present the threads in a very nice way.
https://threadreaderapp.com

And when I added the link given by this site in hoarder, it archived it nicely (I have the full archive enabled).

**Reply by ruhrpotter** [reply]
This would be amazing! As an alternative, would it be possible to bypass the internal analysis somehow and setup a custom script for certain websites?
